File: projecteuler solutions/projectEulerproblem4.js
let maxnum = 999;
let palindrome = 0;
let productChar = 0;
let productRv = 0;
let product = 0;

for (let x = 100; x < maxnum; x++) {
  for (let y = 100; y < maxnum; y++) {
    product = x * y;
    productChar = String(product);
    productRv = productChar.split("").reverse().join("");

    if (productChar == productRv && product > palindrome) {
        palindrome = product;
        console.log(palindrome);
    }
  }
}

console.log(palindrome);

File: projecteuler solutions/projectEulerproblem4.py
maxnum = 999
palindrome = 0
productChar = 0
productRv = 0
product = 0

for x in range(100,maxnum):
    for y in range(100,maxnum):
        product = x * y
        productChar = str(product)
        productRv = str(product)[::-1]

        if (productChar == productRv and product > palindrome):
            palindrome = product
print(palindrome)
           